static/templates/compose.hbs

```diff
@@ -85,6 +85,7 @@
                                 <input type="text" class="recipient_box" name="stream_message_recipient_stream" id="stream_message_recipient_stream" maxlength="30" value="" placeholder="{{t 'Stream' }}" autocomplete="off" tabindex="0" aria-label="{{t 'Stream' }}" />
                                 <i class="fa fa-angle-right" aria-hidden="true"></i>
                                 <input type="text" class="recipient_box" name="stream_message_recipient_topic" id="stream_message_recipient_topic" maxlength="60" value="" placeholder="{{t 'Topic' }}" autocomplete="off" tabindex="0" aria-label="{{t 'Topic' }}" />
+                                <button type="button" class="narrow_to_compose_recipients zulip-icon zulip-icon-arrow-left-circle" aria-label="{{t 'Narrow to compose recipient (Ctrl+.)' }}" data-tippy-content="{{t 'Narrow to compose recipient (Ctrl+.)' }}"></button>
```

Comment:
Can you fix the `tab` order? After pressing `Shift+tab` in the compose textarea, the focus should go to topic / pm input field.

Reply:
The tab order looks correct now.